### PR TITLE
fix: make block info in anchor proof configurable

### PR DESCRIPTION
**Pull request by smrz2001**
I haven't added config options to CAS before so not fully sure if I've covered all the changes. Please let me know if I've missed something.

Also, is this something I should add a unit test for? `anchor-service.test.ts` seemed like the right place, if so.

### CODE REVIEW

File: config/env/dev.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "anchorControllerEnabled": "@@ANCHOR_CONTROLLER_ENABLED",
   "anchorLauncherUrl": "@@ANCHOR_LAUNCHER_URL",
   "expirationPeriod": "@@ANCHOR_EXPIRATION_PERIOD",
+  "includeBlockInfoInAnchorProof": "@@INCLUDE_BLOCK_INFO_IN_ANCHOR_PROOF",
```
Comment (stbrody) [inline config/env/dev.json:8]: is there a clay.json we need to update also?

**Review by stbrody — APPROVED** [review]
lgtm

File: config/env/dev.json (hunk 1 above)
Comment (smrz2001) [reply config/env/dev.json:8]: Good question, there are 3 files: `dev.json`, `prod.json`, and `test.json`. No separate file for Clay.

Comment (smrz2001) [reply config/env/dev.json:8]: Just also checked that previously added config values went into `dev.json` and `prod.json` and not in other places.

**Pull request merged**